misc/azure-release.yml

```diff
@@ -27,14 +27,24 @@ stages:
     steps:
     - task: UsePythonVersion@0
     - template: azure-libtiledb-darwin.yml
+  
+  - job: build1_libtiledb_on_linux
+    container: quay.io/pypa/manylinux2010_x86_64:2021-06-07-00faba2
+
+    steps:
+    - task: UsePythonVersion@0
+    - template: azure-libtiledb-darwin.yml
 
   - job: build2_python
-    dependsOn: build1_libtiledb
+    dependsOn: [build1_libtiledb, build1_libtiledb_on_linux]
```

Comment:
Can we use `dependsOn` within each matrix entry so that we don't block all the builds?

Reply:
https://github.com/microsoft/azure-pipelines-yaml/issues/420

This does not seem to be possible yet.

Reply:
Ok. We could split into separate jobs and use a template for the rest of the steps, but probably not worth it unless we really notice a bottleneck.